src/Uno.UI/Microsoft/UI/Xaml/Controls/CommandBar/CommandBar_Partial.cs

```diff
@@ -0,0 +1,3332 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using DirectUI;
+using Uno.Disposables;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.System;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Uno.UI.Extensions;
+using static Microsoft.UI.Xaml.Controls._Tracing;
+using Uno.UI.Xaml.Input;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml;
+using Popup = Windows.UI.Xaml.Controls.Popup;
+using CommandBarTemplateSettings = Microsoft.UI.Xaml.Controls.Primitives.CommandBarTemplateSettings;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	partial class CommandBar : IMenu
```

Comment:
I'm not sure we've settled on a firm convention, but it seems more common to use the C# naming convention of having the name of the class before the first dot, then indicating the original source file after the first dot and before the file extension. So in this case, something like 'CommandBar.partial.cs'.